src/technology_models/technologies/demand_tech.jl

```diff
@@ -58,7 +58,8 @@ function add_to_expression!(
     mapping_inv = Dict("2030" => 1, "2035" => 2)
 
     time_steps = get_time_steps(container)
-    expression = get_expression(container, T(), PSIP.Portfolio)
+    # expression = get_expression(container, T(), PSIP.Portfolio)
+    expression = add_expression_container!(container, expression_type, D, time_steps)
```

Comment:
Revert this change back to get_expression. It should work with the new dev script from Rodrigo where the template is built